src/app/api/services/dot-messages-service.ts

```diff
@@ -42,7 +42,13 @@ export class DotMessageService {
     }
 
     get(key: string): string {
-        return this.messagesLoaded[key];
+        if (arguments.length > 1) {
```

Comment:
it is not better use something like:

get(key: string, arguments: any[]): string

we have to add doc to this method too

Reply:
We don't need that because [arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments) is a "special JS code.